app/ante/eth.go

```diff
@@ -477,45 +475,38 @@ func (esc EthSetupContextDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simul
 // If fee is high enough or not CheckTx, then call next AnteHandler
 // CONTRACT: Tx must implement FeeTx to use MempoolFeeDecorator
 type EthMempoolFeeDecorator struct {
-	feemarketKeeper evmtypes.FeeMarketKeeper
-	evmKeeper       EVMKeeper
+	evmKeeper EVMKeeper
 }
 
-func NewEthMempoolFeeDecorator(ek EVMKeeper, fmk evmtypes.FeeMarketKeeper) EthMempoolFeeDecorator {
+func NewEthMempoolFeeDecorator(ek EVMKeeper) EthMempoolFeeDecorator {
 	return EthMempoolFeeDecorator{
-		feemarketKeeper: fmk,
-		evmKeeper:       ek,
+		evmKeeper: ek,
 	}
 }
 
 // AnteHandle ensures that the provided fees meet a minimum threshold for the validator,
 // if this is a CheckTx. This is only for local mempool purposes, and thus
 // is only ran on check tx.
+// It only do the check if london hardfork not enabled or feemarket not enabled, because in that case feemarket will take over the task.
```

Comment:
```suggestion
// It only performs the check if London hard fork is not enabled or the fee market param is not enabled, because in that case, the fee market will take over the task.
```